addons/sale_product_configurator/static/tests/product_configurator.test.js

```diff
@@ -226,39 +223,6 @@ QUnit.module('Product Configurator', {
         form.destroy();
     });
 
-    QUnit.test('Select a product in the list and check for template loading', async function (assert) {
-        assert.expect(1);
-
-        const product_configurator_form = await createView({
-            View: ProductConfiguratorFormView,
-            model: 'sale_product_configurator',
-            data: this.data,
-            arch:
-                '<form js_class="product_configurator_form">' +
-                    '<group>' +
-                        '<field name="product_template_id" class="oe_product_configurator_product_template_id" />' +
```

Comment:
test removed because this field is now readonly in the real arch and this feature no longer exists